### PR TITLE
Add pigz to seqtk requirements

### PR DESCRIPTION
**Pull request by mvdbeek**
Since gzip can't be installed via Conda, and not
all versions of gzip support --no-name.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [ ] .shed.yml file ok
    - [ ] Toolshed user `iuc` has access to associated toolshed repo(s)
* [ ] Indentation is correct (4 spaces)
* [ ] Tool version/build ok
* [ ] `<command/>`
  - [ ] Text parameters, input and output files `'single quoted'`
  - [ ] Use of `<![CDATA[ ... ]]>` tags
  - [ ] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [ ] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] Help
  - [ ] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [ ] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

### CODE REVIEW

File: tools/seqtk/macros.xml [hunk 1]
```diff
@@ -24,7 +25,7 @@
         <param name="in_file" type="data" format="fasta,fastq,fasta.gz,fastq.gz" label="Input FASTA/Q file"/>
     </xml>
     <token name="@CONDITIONAL_GZIP_OUT@"><![CDATA[
-    #echo "| gzip --no-name" if $in_file.is_of_type('fasta.gz', 'fastq.gz') else "" # > '$default'
+    #echo "| pigz --no-name --no-time" if $in_file.is_of_type('fasta.gz', 'fastq.gz') else "" # > '$default'
```
[the remark this answers is not in the document]
Comment (mvdbeek) [reply tools/seqtk/macros.xml]: Given that seqtk is exclusively single-thread AFAICT I don't think that we'd gain anything.

[the remark this answers is not in the document]
Comment (bernt-matthias) [reply tools/seqtk/macros.xml]: +1

**Comment by bernt-matthias** [comment]
looks good to me. merge?

**Comment by mvdbeek** [comment]
That would be good, I think!

**Comment by bernt-matthias** [comment]
One last thing:  awk should be added to seqtk_subseq.xml as well. 

One might also think about a token for the awk version, or just add it to the macro since it is used also in quite a lot of the seqtk tools .. but up to you.

**Comment by bernt-matthias** [comment]
> One last thing

how about tool versions? Before it was 1.3.x (`x \in {0,2}`) and now its 1.3+galaxy1. At least lexicographic comparison gives 1.3.x > 1.3+galaxy1 (is this the way version comparison works). Should we stick with 1.3.x for now?

**Comment by nsoranzo** [comment]
> > One last thing
> 
> how about tool versions? Before it was 1.3.x (`x \in {0,2}`) and now its 1.3+galaxy1. At least lexicographic comparison gives 1.3.x > 1.3+galaxy1 (is this the way version comparison works). Should we stick with 1.3.x for now?

I'm working on fixing this.

**Comment by mvdbeek** [comment]
Alright, thanks @nsoranzo @bernt-matthias.

**Comment by bernt-matthias** [comment]
Thanks @nsoranzo and @mvdbeek. I will merge as soon as the tests are done.

**Pull request merged**